lib/SingleSelleckt.js

```diff
@@ -189,21 +189,38 @@ _.extend(SingleSelleckt.prototype, {
         matchTerm = term.toLowerCase();
 
         return _.reduce(items, function(memo, item){
-            var matchIndex;
+            var matchIndex, matchStart, matchEnd;
 
             if (!item.value) {
                 return memo;
             }
 
             matchIndex = item.label.toLowerCase().indexOf(matchTerm);
+            matchStart = matchIndex;
+            matchEnd = matchIndex + (term.length - 1);
+
+            if (matchIndex === -1 && item.aliases && item.aliases.length > 0) {
+                // try the aliases
+                matchIndex = item.aliases.reduce(function(index, alias) {
+                    if (index === -1) {
+                        return String(alias).toLowerCase().indexOf(matchTerm);
+                    } else {
+                        return index;
+                    }
+                }, -1);
+
+                if (matchIndex !== -1) {
+                    matchStart = matchEnd = 0;
```

Comment:
Doesn't `0` mean that a match was found at the first character of the string? I know `matchEnd` will be `0` too, but is there a reason this is `0` and not `-1` or `null` or something else?